taipy/core/data/csv.py

```diff
@@ -122,12 +122,13 @@ def __init__(
             **properties,
         )
         self._path = properties.get(self.__PATH_KEY, properties.get(self.__DEFAULT_PATH_KEY))
+        if self._path and ".data" in self._path:
```

Comment:
`.data`?
Can we at least put it as a const?

Reply:
This is only to migrate old taipy projects to the new structure. If .data is on the datanode current path, we move the file to the new folder. I can put it in a constant.